packages/gatsby-react-router-scroll/src/ScrollContainer.ts

```diff
@@ -16,16 +17,25 @@ const contextTypes = {
   scrollBehavior: PropTypes.object,
 }
 
-class ScrollContainer extends React.Component {
-  constructor(props, context) {
+type Props = PropTypes.InferProps<typeof propTypes>
+type Context = PropTypes.InferProps<typeof contextTypes>
```

Comment:
let's manually type these out. I don't want PropTypes being the source of truth.